### PR TITLE
Proposed improvements for info title rendering.

### PR DESCRIPTION
**Pull request by jbelew**
As I mentioned in the comments for https://github.com/jurialmunkey/skin.arctic.horizon.2/issues/682, here are my proposed changes to the info title element in an effort to make the double line presentation more prominent. The code itself is a bit of a hack giving negative top locations, but it works. 

Here are the justifications --
1. Font sizes align with the existing Pentatonic scale.
2. Slightly smaller info_titles allow more space before wrapping occurs. Cases, where titles wrap, drop drastically and when they do, it's a reasonable decision in testing against my library.  
3. Increased the line spacing so the distance of the info_title baseline and the top of the info line matches the spacing between the info line and the ratings. Allows letter ascenders and descenders more space to reduce the "cramped" feeling.
4. Though the tops of the single and double titles don't align, their baselines do. Affects Info screen rendering slightly, but because you aren't seeing different titles displayed in rapid succession while in the info screen, it's a good tradeoff.
5. Added standard, WCAG-type 3:1 drop shadow to info_titles to increase the prominence and improve readability when fanart impacts the text contrast ratio.

The bottom line is that it's a matter of artistic preference, but give it a try for a bit. I'm so used to it that I can't go back to the old presentation. I've been running the changes for the past month so it's pretty well tested against my library. I've seen no issues with the 64/97 ratio in terms of functional calculation. 

Composite image showing new info_title alignment.
Untitled-1

### CODE REVIEW

**Comment by jbelew** [comment]
FYI ... I didn't play with the Clear Art container. If this moves forward, that container should be shifted slightly to match.

**Comment by jurialmunkey** [comment]
The alignment here is *way* out of bounds. The bounding of the view is needs to be followed:

screenshot00150


Recommendations window title doesn't center align with header row. All header items must have their centre point between the two bold red lines.

screenshot00151


Proper alignment is not only visually important but also for forward planning of new features. Consistent spacing and alignment is necessary to accomodate slide/zoom type animations by providing consistent anchor points. That issue is already highlighted here in the recommendations window and will only cause more issues implementing new features in the future.

**Comment by jbelew** [comment]
Yeah, that's pretty much the response I expected. 

Sometimes grids can lock you into a corner though. Took me two years to get the old Director of Design I used to work with to admit that his 18-column grid system just wasn't working. ;)

**Comment by jurialmunkey** [comment]
This is not about quantising to a grid though. There are plenty of places the skin allows elements to have a looseness that doesn't snap to the grid. It also isn't about filling the space perfectly as even the current implementation doesn't do that.

The issue at hand here is about view boundaries and alignment. These changes cause the double row to fall out of bounds and not provide the required spacing from the header. It also breaks the centre alignment necessary for the recommendations window transition from title to header.

**Comment by jurialmunkey** [comment]
What exactly is the issue that you are trying to address here? I feel like these "solutions" are all putting the cart before the horse.

I'll talk out my thinking process here to give some insight:

- If the issue is mostly that the "double line doesn't look great", then work should be done to make it look *better* instead of compromising the appearance of the single line to sidestep it.
- If the issue is mostly that the single line wraps too early because it only "makes sense visually on long titles", then the most recent skin version already addresses that problem when flixart is disabled by allowing the title to expand to use more of the available horizontal space.

It's impossible to accomodate every possible setup, so I have three user archetypes that I consider when making major changes: (1) visual users (flixart+clearlogo), (2) performance users (fanart+text), and (3) minimalist users (blur+text). 

That means further attempts at improving the single line can be ruled out as bespoke implementations: the most recent skin changes already address the early wrapping issue for performance and minimalist users; and for visual users it is largely moot since they use clearlogos. 

Following these principles, focus should be directed more towards improving the double line.
